### PR TITLE
[0.17] FEM fix peeq import from frd result 

### PR DESCRIPTION
**Pull request by berndhahnebach**
- [x] Branch rebased on latest master `git pull --rebase upstream master`
- [x] Unit tests confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [x] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)

### CODE REVIEW

File: src/Mod/Fem/feminout/importCcxFrdResults.py [hunk 1]
```diff
@@ -388,44 +415,44 @@ def readResult(frd_input):
...
+            # strain_5 = float(line[61:73])
+            # strain_6 = float(line[73:85])
             mode_strain[elem] = FreeCAD.Vector(strain_1, strain_2, strain_3)
 
+        # Check if we found a equivalent plastic strain section
         if line[5:7] == "PE":
             mode_peeq_found = True
-        # we found an equivalent plastic strain line in the frd file
         if mode_peeq_found and (line[1:3] == "-1"):
+            # we found a equivalent plastic strain line
```
Comment (luzpaz) [inline src/Mod/Fem/feminout/importCcxFrdResults.py]: `s/found a/found an/`

File: src/Mod/Fem/feminout/importCcxFrdResults.py [hunk 2]
```diff
@@ -446,82 +475,120 @@ def readResult(frd_input):
...
-                mode_results['mflow'] = mode_massflow
-                mode_results['npressure'] = mode_networkpressure
-                mode_results['time'] = timestep
-                results.append(mode_results)
-                mode_massflow = {}
-                mode_networkpressure = {}
-                eigenmode = 0
-            nodes_found = False
-            elements_found = False
+        # on changed --> write changed values in mode_result --> will be the first to do on a empty mode_result
```
Comment (luzpaz) [inline src/Mod/Fem/feminout/importCcxFrdResults.py]: s/on a/on an/

File: src/Mod/Fem/feminout/importCcxFrdResults.py [hunk 3]
```diff
@@ -446,82 +475,120 @@ def readResult(frd_input):
...
+            mode_results['number'] = eigenmode
+            mode_eigen_changed = False
+
+        if mode_time_changed:
+            mode_results['time'] = timestep
+            # mode_results['time'] = 0  # Don't return time if static  # WARUM?
+            mode_time_found = False
+            mode_time_changed = False
+
+        # here we are in the indent of loop for every line in frd file, do not add a print her :-)
```
Comment (luzpaz) [inline src/Mod/Fem/feminout/importCcxFrdResults.py]: `s/print her/print here/`

**Comment by berndhahnebach** [comment]
all of luzpaz fixed

**Comment by yorikvanhavre** [comment]
merged

**Pull request closed without merging**